### PR TITLE
docs(Api): fix holocronMoudule examples

### PR DESCRIPTION
**Pull request by infoxicator**
- Moved deprecated method to the bottom of the list to avoid accidental usage
- Updated examples and remove usages of `load`

### CODE REVIEW

File: packages/holocron/API.md [hunk 1]
```diff
@@ -245,7 +189,7 @@ An action creator that loads Holocron modules and their data.
 ```js
 import { composeModules } from 'holocron';
 
-const load = (props) => (dispatch) => dispatch(composeModules([
+const loadModuleData = (props) => (dispatch) => dispatch(composeModules([
```
Comment (mtomcal) [inline packages/holocron/API.md:192]: This needs to be refactored as `loadModuleData` returns `void` not a `thunk`. 

```js
({ store: { dispatch }, ownProps }) => dispatch(composeModules( //...
```

Comment (infoxicator) [reply packages/holocron/API.md:192]: 😅 thanks! that one escaped me... I should have done the generator PR before this one!

Comment (infoxicator) [reply packages/holocron/API.md]: fixed on https://github.com/americanexpress/holocron/pull/27/commits/db59438903e66e352ceabe081e6e3648a384bde9

**Comment by infoxicator** [comment]
@mtomcal in the new holocron api are we still using `options: { ssr: true}` ? do you have the example on how to enable SSR I will add it to this PR? 🙇‍♂️

**Comment by mtomcal** [comment]
@infoxicator We no longer need to have `options: { ssr: true}` unless you use deprecated methods.

File: packages/holocron/API.md (hunk 1 above)
Comment (mtomcal) [reply packages/holocron/API.md]: Actually `loadModuleData` returns `Promise<void>` technically my bad. Looks good

**Comment by mtomcal** [comment]
@infoxicator Merged #28. Would you make sure that the conflicted docs sections have the `ONE-DOCS-ID` wrapping appropriate sections? Thanks!

**Pull request merged**